internal/controller/notebook_controller.go

```diff
@@ -424,3 +442,37 @@ func (r *NotebookReconciler) notebookPVC(nb *apiv1.Notebook) (*corev1.Persistent
 
 	return pvc, nil
 }
+
+func (r *NotebookReconciler) callSignedUrlGenerator(notebook *apiv1.Notebook) (string, error) {
+	// TODO(bjb): we should be using TLS here
+	conn, err := grpc.Dial(
```

Comment:
Establish connection and client in `main()` and pass into Reconciler as field

Reply:
np. I should have put a note that I planned to circle back on this after tests were working